lib/deployinator/controller.rb

```diff
@@ -147,7 +147,7 @@ def run(options, requires_lock = true)
       elsif options[:method].match(/force_builda/)
         env = "force asset rebuild"
       else
-        env = options[:method][/(dev|qa|production|princess|prod|webs|stage|staging|config)/i, 1] || "other"
+        env = options[:method][/(dev|qa|production|princess|preprod|prod|webs|stage|staging|config)/i, 1] || "other"
```

Comment:
Shouldn't we be prefixing all of those with a `^` so that we're only checking for names that start a certain way?

Reply:
does it really hurt anything not to? curious on the implication... like a broader match can't really hurt here right?

Reply:
On the next line down, it does affect us so maybe not necessary here but the line below needs to handle `preprod` somehow.